### DISCUSSION TITLE
Specific rules for openHAB Add-ons maintainers

### DISCUSSION THREAD

**Discussion by kaikreuzer** [openhab/openhab-addons#14694] [Add-on Maintainers Discussions]
Besides the [global rules for maintainers](https://github.com/openhab/openhab-distro/discussions/1505), we have the following additional agreements for openHAB 2 Add-ons:

- PRs are always checked for compliance with the [coding guidelines](https://www.openhab.org/docs/developer/guidelines.html).
- The [review checklist](https://github.com/openhab/openhab-addons/wiki/Review-Checklist) should additionally be used as a guidance to catch common mistakes during a review.
- Review priority should be on (1) consistency towards the user (e.g. consistent Thing modelling, proper README) and on (2) architecture consistency (correct use of openHAB APIs, use of provided features such as shared HTTP client, system channel types, etc.) and on (3) consistent runtime behavior (like correct Thing status changes, clean shutdown and proper logging) and not so much on individual code style.
- PRs can be merged by any maintainer.

**Comment by nelsonaponte** [comment]
Hi, @kaikreuzer.

The first two links are broken. Is that intentional?
I came to this page from https://www.openhab.org/docs/developer/governance.html

**Reply by lsiepel** [reply]
I have updated both links:
https://github.com/openhab/openhab-distro/discussions/1505
https://www.openhab.org/docs/developer/guidelines.html
